apps/fhir/bluebutton/views/search.py

```diff
@@ -54,7 +54,13 @@ def get(self, request, *args, **kwargs):
         return super().get(request, self.resource_type, *args, **kwargs)
 
     def build_url(self, resource_router, resource_type, *args, **kwargs):
-        return resource_router.fhir_url + resource_type + "/"
+        api_ver = kwargs.get('ver')
+        if resource_router.fhir_url.endswith('v1/fhir/'):
+            # only if called by tests
+            return "{}{}/".format(resource_router.fhir_url, resource_type)
+        else:
+            return "{}/{}/fhir/{}/".format(resource_router.fhir_url, api_ver,
+                                           resource_type)
 
 
 class SearchViewPatient(SearchView):
```

Comment:
Example continued.

Would add `version` as an instance variable for the `SearchViewPatient` class here.


```
class SearchViewPatient(SearchView):
    # Class used for Patient resource search view
    version = None

    def __init__(self, version=1):
        self.version = version
        self.resource_type = "Patient"
```

Could default the version = 1, or require it to be passed in with:

```
def __init__(self, version):
```